### PR TITLE
base engine interface

### PR DESCRIPTION
**Pull request by anakwaboe4**

### CODE REVIEW

File: backend/TikTakToe.Engines/Class1.cs
Comment (EwoutHeb) [inline backend/TikTakToe.Engines/Class1.cs:1]: delete plz

File: backend/TikTakToe.Core/Enums/Engines.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace TikTakToe.Core.Enums
+{
+    public enum Engines
+    {
+        Player = 0,
+        PerfectOptemism = 1,
+
```
Comment (EwoutHeb) [inline backend/TikTakToe.Core/Enums/Engines.cs]: Remove empty line

File: backend/TikTakToe.Engines/Class1.cs
Comment (anakwaboe4) [reply backend/TikTakToe.Engines/Class1.cs:1]: updated

File: backend/TikTakToe.Core/Enums/Engines.cs (hunk 1 above)
Comment (anakwaboe4) [reply backend/TikTakToe.Core/Enums/Engines.cs]: done

**Pull request merged**